src/Sulu/Bundle/CoreBundle/Content/templates/internal-link.xml

```diff
@@ -19,7 +19,7 @@
             </meta>
         </property>
 
-        <property name="internal_link" type="single_internal_link">
```

Comment:
@danrot I don't know 🙈

Reply:
@wachterjohannes Do you have an idea?

Reply:
What I tried is to delete this file, and I was still able to create an internal link of a page, and the redirect on the website still worked. I assumed that this would be somehow connected to this part of the system. Are these just leftovers?